### PR TITLE
fix:Missing double quote in Build a Telephone Number Validator

### PR DESCRIPTION
**Pull request by sureshkrishnan-v**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #55270

### CODE REVIEW

**Comment by ilenia-magoni** [comment]
> Closes #XXXXX

please update this to link to the issue

**Comment by sureshkrishnan-v** [comment]
> > Closes #XXXXX
> 
> please update this to link to the issue

I cannot understand what to do mam

File: curriculum/challenges/arabic/15-javascript-algorithms-and-data-structures-22/build-a-telephone-number-validator-project/build-a-telephone-number-validator.md
Comment (naomi-lgbt) [inline curriculum/challenges/arabic/15-javascript-algorithms-and-data-structures-22/build-a-telephone-number-validator-project/build-a-telephone-number-validator.md:1]: You have changed the Arabic file. You should only change the English file.

**Comment by ilenia-magoni** [comment]
> > > Closes #XXXXX
> > 
> > 
> > please update this to link to the issue
> 
> I cannot understand what to do mam

replace as instructed by the comment above that line

**Pull request closed without merging**